src/Cosi-BLS/block.lisp

```diff
@@ -176,7 +176,7 @@ added to the blockchain."
       (setf timestamp (- (get-universal-time) *unix-epoch-ut*))
```

Comment:
make block construction code responsible for internal content format, instead of caller